Makefile

```diff
@@ -568,7 +572,7 @@ $(STATIC_NAME): $(OBJS) | $(LIB_BUILD_DIR)
 	@ echo AR -o $@
 	$(Q)ar rcs $@ $(OBJS)
 
-$(BUILD_DIR)/%.o: %.cpp | $(ALL_BUILD_DIRS)
+$(BUILD_DIR)/%.o: %.cpp | $(ALL_BUILD_DIRS) $(CUB_DIR)
```

Comment:
Removed this one and left only %.cu rule - it get's triggered when NVCC is running.